tests/phpunit/CRM/Event/Form/SelfSvcUpdateTest.php

```diff
@@ -30,7 +30,8 @@ public function testForm(): void {
     $mut->checkAllMailLog([
       'Your Event Registration has been cancelled',
       'Annual CiviCRM meet',
-      'Registration Date: February 19th, 2007',
+      'Registration Date',
+      'February 19th, 2007',
```

Comment:
Ok you're gonna have to work for this one. Now two lines down it doesn't like it since it doesn't have the weekday.

Reply:
@demeritcowboy yeah - I suspect the 10 templates in this PR - https://github.com/civicrm/civicrm-core/pull/28704 don't have test cover or I would be doing that there :-(

Reply:
This looks like it got updated to be a different PR now?

Reply:
argh - that follow on commit is not ready yet - hopefully I removed it correctly